### PR TITLE
Implement programmatic api

### PR DESCRIPTION
**Pull request by timneutkens**
Moved the website to website.js. Website.js should move to a seperate package (with the website folder ofcourse)

### CODE REVIEW

**Comment by timneutkens** [comment]
Also this package (now-logs-server) can be published to npm. Programmatic usage:
`require('now-logs-server/server')(io)` where `io` is a socket.io instance.

File: index.js [hunk 1]
```diff
@@ -1,28 +1,14 @@
+const { NOW_LOGS_SECRET_KEY, NOW_LOGS_PORT = 3000 } = process.env;
+
 if (process.env.NOW_LOGS_SECRET_KEY) {
```
Comment (berzniz) [inline index.js]: Can be simplified to:
```
if (NOW_LOGS_SECRET_KEY) {
```

File: package.json [hunk 2]
```diff
@@ -3,14 +3,17 @@
   "version": "1.0.0",
   "description": "",
   "main": "index.js",
+  "files": [
```
Comment (berzniz) [inline package.json:6]: Can you clarify what this does exactly? Does it only apply to `now.sh` service?

File: website.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
+const { NOW_LOGS_SECRET_KEY, NOW_LOGS_PORT = 3000 } = process.env;
+
+if (process.env.NOW_LOGS_SECRET_KEY) {
```
Comment (berzniz) [inline website.js:3]: Can be simplified to:
```
if (NOW_LOGS_SECRET_KEY) {
```

File: yarn.lock [hunk 4]
```diff
@@ -0,0 +1,568 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```
Comment (berzniz) [inline yarn.lock]: While `yarn` is cool, I'd leave this out and keep only the `npm` version in. This makes it too opintionated

**Review by berzniz — CHANGES_REQUESTED** [review]
I think this is a good change overall, but it's a lot of changes. I added my comments inline.

The website should also be split into its own repo and a `now-logs-server` npm package should be published, I will do this later

File: package.json (hunk 2 above)
Comment (timneutkens) [reply package.json:6]: It applies to both, so good catch, added `server.js`.

`website.js` and the `website` folder should be moved into a seperate repo 👍

**Review by berzniz — APPROVED** [review]

**Pull request merged**